test/ui-automation/test/helpers/wallet.js

```diff
@@ -93,6 +93,10 @@ exports.validateUserInput = async (vaultName, errMsg) => {
   await _validateUserInput(vaultName, errMsg);
 };
 
+exports.validateRenameInput = async (credentialName, errMsg) => {
```

Comment:
Can we remove `input` suffix from function names and have a concise function name which describes what it does ?